src/backend/TrafficCourts/TrafficCourts.Staff.Service.Test/Controllers/DocumentControllerTest.cs

```diff
@@ -222,15 +205,16 @@ public async void TestDownloadDocument200Result()
     {
         // Arrange
         var fileStream = new MemoryStream(System.Text.Encoding.ASCII.GetBytes("FileData"));
-        Coms.Client.File mockFile = new(fileStream, "testFile");
+        Coms.Client.File file = new(fileStream, "testFile");
         var comsService = new Mock<IStaffDocumentService>();
         Guid guid = Guid.NewGuid();
-        mockFile.Metadata.Add("ticket-number", "AO38375804");
-        mockFile.Metadata.Add("virus-scan-status", "clean");
-        var filename = mockFile.FileName;
+        DocumentProperties properties = new DocumentProperties { TcoDisputeId = 1 };
+        //file.Metadata.Add("ticket-number", "AO38375804");
+        //file.Metadata.Add("virus-scan-status", "clean");
```

Comment:
should these lines be deleted?

Reply:
Probably, seems the test doesnt use them as it still passes.